tests/Doctrine/Tests/DBAL/Functional/Schema/PostgreSqlSchemaManagerTest.php

```diff
@@ -502,6 +502,34 @@ public function autoIncrementTypeMigrations() : array
             'bigint->int' => ['bigint', 'integer', 'INT'],
         ];
     }
+
+    /**
+     * @group 2925
+     */
+    public function testAlterTableChangePrimaryKey() : void
+    {
+        $tableFrom = new Table('primary_key_userid');
+        $column    = $tableFrom->addColumn('user_id', 'integer');
+        $column->setAutoincrement(true);
+        $tableFrom->setPrimaryKey(['user_id']);
+        $this->_sm->dropAndCreateTable($tableFrom);
+
+        $tableFrom = $this->_sm->listTableDetails('primary_key_userid');
+        self::assertTrue($tableFrom->getColumn('user_id')->getAutoincrement());
+
+        $tableTo = new Table('primary_key_id');
+        $column  = $tableTo->addColumn('id', 'integer');
+        $column->setAutoincrement(true);
+        $tableTo->setPrimaryKey(['id']);
+
+        $c    = new Comparator();
+        $diff = $c->diffTable($tableFrom, $tableTo);
+        self::assertInstanceOf(TableDiff::class, $diff, "There should be a difference and not false being returned from the table comparison");
```

Comment:
Constant strings should be enclosed in `'`.

Reply:
> There should be a difference and not false

It's a very vague assertion. I'd omit it together with the previous one about the autoincrement and only focus on the end result where the new primary key is `(id)`.

Reply:
No sniff I'm aware of, but shouldn't be hard to implement. Just needs to be compatible i.e. with single statements:
```php
if (...) {
    continue;
}
```
or comments:
```php
foo();

// comment
return 123;
```